editor/register_editor_types.cpp

```diff
@@ -128,8 +98,46 @@
 #include "editor/shader/shader_file_editor_plugin.h"
 #include "editor/translations/editor_translation_parser.h"
 #include "editor/version_control/editor_vcs_interface.h"
+
+// 2D.
+#include "editor/scene/2d/camera_2d_editor_plugin.h"
```

Comment:
Most of this PR is grouping 3D, but I also grouped 2D includes here, for symmetry with the 3D includes.